server/main.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	"cloud.google.com/go/firestore"
```

Comment:
Using the firestore client library directly here as gocloud's docstore package had several limitations:

- No querying by "array-contains".
- No cursor support.